spec/jobs/reindex_model_job_spec.rb

```diff
@@ -1,11 +1,26 @@
 # frozen_string_literal: true
 
 RSpec.describe HykuAddons::ReindexModelJob do
-  let(:account) { create(:account) }
-  let(:work) { create(:work) }
+  let(:account) { create(:account, cname: cname) }
+  let(:work) { create(:work, doi: []) }
+  let(:second_work) { create(:work, title: ["another work"], doi: []) }
+  let(:prefix) { "10.1234" }
+  let(:cname) { "123abc" }
+  let(:response_body) { File.read(HykuAddons::Engine.root.join("spec", "fixtures", "doi", "mint_doi_return_body.json")) }
+  let(:options) { { cname_doi_mint: [account.cname], use_work_ids: [work.id] } }
 
   before do
-    allow(work.class).to receive(:reindex_everything)
+    GenericWork.all.map(&:destroy)
```

Comment:
Why do we have to destroy all GenericWorks? Why have any objects been created for this test that we don't want created?

Reply:
FYI, `GenericWork.destroy_all` would be a slightly nicer way of writing this.

Reply:
I just checked to make sure ActiveFedora implemented it: 

```
2.7.4 :001 > GenericWork.methods.include?(:destroy_all)
 => true
```

Reply:
@thenapking that ought to be taken out, I was checking something locally and just forgot to take off.